contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,27 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
+Ids can be specified in three different forms:
+- Id only: `#2000`
+- Repo and id: `zulip#2000`
+- Owner and repo and id `zulip/zulip#2000`
+
+Both the username mention and the id can occur at any time in the message. You
+can also mention multiple ids in a single message.
+
+The bot *requires* a default owner and repo to be configured.
+The configuration file should be located at `~/.contrib_bots/github_detail.ini`.
```

Comment:
I decided on that because I thought it made sense to have all bot config files in the same place (If this gets accepted I'll change the other bots to use the same folder.) I used a dot folder partly because I think config files should be hidden, and partly because it was the folder name used as an example by Steve Howell.